### ISSUE TITLE
[Feature]: Display two curves in one graph?

### ISSUE THREAD

**Issue by Freed-Wu** [wandb/wandb#4449]
### Description

```python
#!/usr/bin/env python
"""main."""
import wandb
from torch.utils.tensorboard.writer import SummaryWriter

model = image_models["mbt2018"](pretrained=True, quality=6)

log_dir = '/output/logs'

name = '5'
wandb.init(project="my-test-project-2", dir=log_dir, name=name, id=name,
           config={"learning_rate": 0.001, "epochs": 100, "batch_size": 128},
           sync_tensorboard=True)

writer = SummaryWriter(log_dir=log_dir)

writer.add_scalars('metrics/loss', {'train': 20, 'val': 100}, 0)
writer.add_scalars('metrics/loss', {'train': 10, 'val':  50}, 1)

# Optional
wandb.finish()
```

Can it display curves of same tag to one graph?

This is wandb:

![screen-2022-11-04-22-44-28](https://user-images.githubusercontent.com/32936898/200003372-763ad8bf-0cf8-457a-baa9-ab46b3f246da.jpg)

This is tensorboard:

![screen-2022-11-04-22-44-06](https://user-images.githubusercontent.com/32936898/200003355-37555673-8831-4938-8c53-2db58679cfba.jpg)

### Suggested Solution

/

**Comment by ramit-wandb** [comment]
Hi @Freed-Wu,

Yup, this is already possible. You can read about it in our docs here : https://docs.wandb.ai/ref/app/features/panels/line-plot#compare-two-metrics-on-one-chart

**Comment by Freed-Wu** [comment]
Can it support tensorboard's `add_scalars`?

**Comment by ramit-wandb** [comment]
Hey @Freed-Wu,

Yes, `add_scalar` from tensorboard is supported.

**Comment by Freed-Wu** [comment]
I means Can it display multi curves in one graph like `add_scalars`.

**Comment by Freed-Wu** [comment]
That is how can wandb display similar result with tensorboard? <https://github.com/wandb/wandb/issues/4449#issue-1436190950>

**Comment by ramit-wandb** [comment]
Hey @Freed-Wu,

As I mentioned before, multiple curves are already supported. You will have to do some manual work to get these exactly the way you need them however. Here is a video to aid you with how to do this for any 2 plots:



https://user-images.githubusercontent.com/93002515/201744085-43405125-a054-41ff-886f-9793dfe4fb55.mov

**Comment by ramit-wandb** [comment]
​Hi @Freed-Wu, since we have not heard back from you we are going to close this request. If you would like to re-open the conversation, please let us know!
​

**Comment by oshapio** [comment]
Is it possible to do this automatically? Manually fiddling isn't ideal when many metrics are logged. It also liters workspace and requires deleting 'standalone', ungroupped graphs. It would be nice if logging a dict (with maybe additional flag `group` to not break things) would group things automatically as OP showed.

**Comment by fiskrt** [comment]
I agree that this should be done automatically, since it's impractical and error prone to do this for every single figure. Especially when dealing with more than two curves. **Suggested solution**: copy tensorboard's functionality.

**Comment by OliEfr** [comment]
I would also agree that it would be nice to do this automatically.

**Comment by garrett361** [comment]
Is there an update on this?  I would also love this functionality.

This is also already done for system metrics like GPU power usage for multi-gpu runs. Can't we do the same for other metrics? 

<img width="582" alt="Image" src="https://github.com/user-attachments/assets/20f8e187-a965-4e5c-83c2-2b828d3844a4" />

I tried to look in the `wandb` code to see how these system metrics are reported, but couldn't see anything useful for reporting other metrics similarly. Also not sure if these `System` metrics are just special-cased in the UI.

CC @kptkin 

Related: https://github.com/wandb/wandb/issues/3035

**Comment by olliepro** [comment]
the vega chart options are frankly weak options. The system metrics plots are great examples of what we want to be able to do programatically.

**Comment by blefaudeux** [comment]
wandb not doing anything about that after years of people asking for it & using it on internal plots is disheartening
